src/main/java/inspired/pdf/unbox/elements/TextCell.java

```diff
@@ -69,8 +69,16 @@ public float innerHeight(Bounds viewPort) {
         return lines.length * lineHeight + LINE_PADDING;
     }
 
+    @Override
+    public void setValue(Object value) {
```

Comment:
No, it's responsibility of the TableCell to render the value it's been given. This allows to simplify the way users can provide the data. One could e.g. use a TableCell renderer that prints an picture or in case of a more complex case, one could define a custom TableCell that renders information about a whole entity (e.g. a location that consists of things like building, floor, room etc.). This way, the user won't have to do the conversion manually for each item when rendering the table, but instead just pass the entity as value in a row:

```java
        // Could come directly from database or some other source
        var location = new Location("Hamburg", "Am Hafen", "1", "3 OG", "351");
        ...
        tableModel.addDefaultCell(Location.class, new LocationTableCell());
        ...
        // Use the entity directly when defining the table values
        table.addRow("String field", location, 14.7f);

```